defaults/osdefs/osdefs_darwin.go

```diff
@@ -0,0 +1,10 @@
+// +build darwin
+
+package osdefs
+
+const (
+	// PrCapbsetDrop is prctl PR_CAPBSET_READ argument value
+	PrCapbsetDrop = 0x18
```

Comment:
this seems like the same `syscall` pkg constant as for `linux`? https://golang.org/pkg/syscall/#pkg-constants